source/authentication.txt

```diff
@@ -118,8 +118,9 @@ User Metadata
 Each authentication provider can associate metadata fields with an 
 application user. Some providers, such as Email/Password, always add 
 specific fields. Others allow you to configure the data to associate with 
-each user. Authentication providers include these metadata fields in a 
-user object:
+each user.
+
+To learn more, see :ref:`Authentication Provider Metadata <auth-provider-metadata>`
```

Comment:
```suggestion
To learn more, see :ref:`Authentication Provider Metadata <auth-provider-metadata>`.
```

Reply:
not sure what the deal is, but when I click the link from the page, it takes me to the Enable User Data heading, but if I paste in the same URL, I get to the Authentication Provider Metadata heading 🤷‍♀️

Reply:
Not sure what's happening either but probably a transient staging bug. Nothing in the source would make it point there so here's hoping it goes away in prod :)